lib/renderer/chrome-api.js

```diff
@@ -199,4 +199,6 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
     setPopup () {},
     getPopup () {}
   }
+
+  chrome.i18n = require('./extensions/i18n.js')
```

Comment:
@zcbenz Are you okay breaking this out to a separate file so `chrome-api` doesn't grow too large?